src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,36 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.info("Method completeOrder was called");
+        if (userId == null) {
+            logger.error("Incoming data for userId is null");
+        }
         List<Product> products = getAllProductsFromShoppingCart(userId);
         Order order = new Order(products, userId);
         // NOTE: In production ready code this order identifier should be generated by DB
         // For test purpose we simplify this and return dummy data
         order.setOrderId(1L);
+        logger.info("Returning order ID : {}", order.getOrderId());
         return order;
     }
 
     private List<Product> getAllProductsFromShoppingCart(Long userId) {
+        logger.info("Method getAllProductsFromShoppingCart was called");
         // NOTE: In production ready code this method should fetch data from DB
         // For test purpose we simplify this method and return dummy data
         Product iphone = new Product("iPhone X", BigDecimal.valueOf(1199));
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.info("Returning list of All products for userId : {}", userId);
```

Comment:
The TODO: was about to log the fact the method has been called )
But I don't know if it matters so it's up to you =)

Reply:
I got  logger.info("Method getAllProductsFromShoppingCart was called"); at the line above)

Reply:
Also about your config xml file: as I see, we don't have any System.out,println() to replace with the corresponding logger message, so do we really need configuration for the Console out. IDK, just a suggestion =)

Reply:
agree, the message should be about succesfull fetch from the DB, and not about the method call or that it returns some value, so let's do that

Reply:
> Also about your config xml file: as I see, we don't have any System.out,println() to replace with the corresponding logger message, so do we really need configuration for the Console out. IDK, just a suggestion =)

But its in the task itself Configure the logger in this project. Add appenders for File and Console